start.sh

```diff
@@ -53,6 +53,9 @@ fi
 if [ -n "$(dig A +short nextcloud-aio-stalwart)" ] && ! grep -q "mail.{\$NC_DOMAIN}" /Caddyfile; then
     cat << CADDY >> /Caddyfile
 https://mail.{\$NC_DOMAIN}:443 {
+    # import GEOFILTER
+    reverse_proxy nextcloud-aio-stalwart:8080
```

Comment:
See https://github.com/nextcloud/all-in-one/pull/4556/files#r1575901974

Reply:
Change to 10003